### ISSUE TITLE
Naming scheme for channel ranges

### ISSUE THREAD

**Issue by tjgalvin** [tjgalvin/flint#194]
At the moment I am using the following naming scheme to indicate channel ranges when forming images with only a subset of channels:

```
SB59058.RACS_1626-84.beam18.round4.i.ch96-97
```

This would be supplied to the imager (wsclean -name) and output products would be formed around this prefix (e.g. the ImageSet will have its prefix set to this). 

Two problems:
- the scheme itself may not be the best. These should at the very least be zero-padded. There is a question around whether we want a range, or simply have the starting index. My feeling is the channel range may be useful in the future should we want to image multiple channels at a time, instead of a single. Same thought around doing the time-range imaging
- The regex's that are used to separate the wsclean added naming components, and the general flint name parser both need to be expanded to recognise this `chNNNN-NNNN` field, should it be present. I believe it is this point that is creating files with names like:
```
SB59058.RACS_1626-84.beam18.round4.i.ch96-97-image.optimal.image.fits
```

in the spectral line workflow

**Comment by tjgalvin** [comment]
@AlecThomson may have strong feelings on point 1.

**Comment by tjgalvin** [comment]
Addressed as part of #196
